src/handlers.pxi

```diff
@@ -66,6 +66,9 @@ async def fuse_lookup_async (_Container c, name):
             c.parent, name, ctx)
     except FUSEError as e:
         ret = fuse_reply_err(c.req, e.errno)
+    except BaseException as e:
+        fuse_reply_err(c.req, errno.EIO)
+        raise
```

Comment:
Will such exceptions ever make to sys.excepthook, or will the be silently swallowed when execution goes from Python to C? In the latter case, I think it would make more sense to call a short helper method here that logs the exception as well as calling fuse_reply_err.

Reply:
I'll test that. Thanks for the suggestion.